ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -463,6 +472,22 @@ private static Format parseFormat(final String formatString) {
         }
       }
 
+      private static Character parseValueDelimiter(final String valueDelimiterString) {
```

Comment:
Where we can, can we avoid the use of `null`s.  Instead make the fact it's optional part of the API and return `Optional<Character>`.

Reply:
If returning null it makes this method easier to use from the arg builder, otherwise I would have just have to do add some code that converts Optional.empty() to null.

Reply:
We aim to avoid `nulls` in our code base, preferring the more explicit use of `Optional`, which explicitly codifies the nullability of a param, return value, etc.

Reply:
Again, I think this should be part of a DataGen refactoring